scipy/spatial/distance.py

```diff
@@ -931,6 +933,57 @@ def kulsinski(u, v, w=None):
     return (ntf + nft - ntt + n) / (ntf + nft + n)
 
 
+def kulczynski1(u, v, w=None):
+    """
+    Compute the Kulczynski 1 dissimilarity between two boolean 1-D arrays.
+
+    The Kulczynski 1 dissimilarity between two boolean 1-D arrays `u` and `v`,
+    is defined as
+
+    .. math::
+
+         \\frac{c_{TT}}
+              {c_{FT} + c_{TF}}
+
+    where :math:`c_{ij}` is the number of occurrences of
+    :math:`\\mathtt{u[k]} = i` and :math:`\\mathtt{v[k]} = j` for
+    :math:`k < n`.
+
+    Parameters
+    ----------
+    u : (N,) array_like, bool
+        Input array.
+    v : (N,) array_like, bool
+        Input array.
+    w : (N,) array_like, optional
+        The weights for each value in `u` and `v`. Default is None,
+        which gives each value a weight of 1.0
+
+    Returns
+    -------
+    kulczynski1 : float
+        The Kulczynski 1 distance between vectors `u` and `v`.
+
+    Examples
```

Comment:
Version added and see also. But is `kulsinski` related to `kulczynski1`? Is there a typo somewhere? If related, maybe add a note about the difference.

```suggestion
        The Kulczynski 1 distance between vectors `u` and `v`.

    .. versionadded:: 1.8.0

    See Also
    --------
    kulsinski

    Examples
```

Reply:
Ok forget about the see also section. I just read the issues and no we do not want to reference a deprecated thing.

Reply:
Can you also add a reference for the metric?